Gaming The Syllabus/Assets/Scripts/Combat/CombatManager.cs

```diff
@@ -14,13 +14,16 @@ public enum GameState {
 
 public class CombatManager : Singleton<CombatManager>
 {
-    private static CombatUnit player;
+    public static GameState gameStatus;
+    public static CombatUnit player;
     private static CombatUnit[] enemies;
 
     private static List<CombatUnit> allCombatants;
 
     private static int indexOfCombatantWithCurrentTurn;
 
+    public static bool isDone = false; // Checks if this script is done running.
```

Comment:
Some scary publics in here

Reply:
Doesn't matter too much since it's a singleton but hey, might as well make them private